### PR TITLE
chore(macro): update l10n-zh translation for {{DeprecatedGeneric}} macro

### PR DESCRIPTION
**Pull request by yin1999**
## Summary

Update l10n-zh translation for `{{DeprecatedGeneric}}` macro.

## Screenshots

image

### CODE REVIEW

**Comment by yin1999** [comment]
poke @irvin for a review

File: kumascript/macros/DeprecatedGeneric.ejs [hunk 1]
```diff
@@ -113,8 +113,8 @@ var str_desc = mdn.localString({
...
   "pl":    "This feature has been removed from the Web standards. Though some browsers may still support it, it is in the process of being dropped. Do not use it in old or new projects. Pages or Web apps using it may break at any time.",
   "ru":    "Эта возможность была удалена из веб-стандартов. Хотя некоторые браузеры по-прежнему могут поддерживать её, она находится в процессе удаления. Не используйте её ни в старых, ни в новых проектах. Страницы или веб-приложения, использующие её, могут в любой момент сломаться.",
-  "zh-CN": "该特性已经从 Web 标准中删除，虽然一些浏览器目前仍然支持它，但也许会在未来的某个时间停止支持，请尽量不要使用该特性。",
-  "zh-TW": "This feature has been removed from the Web standards. Though some browsers may still support it, it is in the process of being dropped. Do not use it in old or new projects. Pages or Web apps using it may break at any time."
+  "zh-CN": "不再推荐使用该特性。虽然一些浏览器仍然支持它，但也许已从相关的 web 标准中移除，也许正准备移除或出于兼容性而保留。请尽量不要使用该特性，并更新现有的代码；参见本页面底部的<a href='#浏览器兼容性'>兼容性表格</a>以指导你作出选择。请注意，该特性随时可能无法正常工作。",
+  "zh-TW": "不再推薦使用此功能。雖然一些瀏覽器仍然支援它，但也許已從相關的 web 標準中移除，也許正準備移除或出於兼容性而保留。請盡量不要使用該功能，並更新現有的代碼；參見本頁面底部的<a href='#瀏覽器相容性'>相容性表格</a>以指導你作出選擇。請注意，該功能隨時可能無法正常工作。"
```
Comment (irvin) [inline kumascript/macros/DeprecatedGeneric.ejs:117]:
```suggestion
  "zh-TW": "不推薦使用此功能。雖可能有一些瀏覽器仍然支援它，但也許已自相關的網頁標準中移除、正準備移除、或僅為了維持相容性而保留。避免使用此功能，盡可能更新現有程式；請參考頁面底部的<a href='#瀏覽器相容性'>相容性表格</a>來下決定。請注意：本功能可能隨時停止運作。"
```

**Review by irvin — CHANGES_REQUESTED** [review]

File: kumascript/macros/DeprecatedGeneric.ejs (hunk 1 above)
Comment (irvin) [reply kumascript/macros/DeprecatedGeneric.ejs:117]: @yin1999 please commit the change, thanks

**Comment by yin1999** [comment]
Ready to merge.

**Comment by yin1999** [comment]
it's time to poke @mdn/mdn-community-engagement

**Review by irvin — APPROVED** [review]

**Comment by caugner** [comment]
Thanks @yin1999 and @irvin! 👏

**Pull request merged**